docs-website/docusaurus.config.js

```diff
@@ -45,16 +45,22 @@ module.exports = {
       },
       items: [
         {
-          to: "docs/",
-          activeBasePath: "docs",
+          type: 'docSidebar',
+          position: 'right',
+          sidebarId: 'overviewSidebar',
           label: "Docs",
-          position: "right",
         },
         {
-          to: "/integrations",
-          activeBasePath: "integrations",
+          type: 'docSidebar',
+          position: 'right',
+          sidebarId: 'integrationsSidebar',
```

Comment:
the existing link shows up without a nav bar https://docs-website-gbp50y504-acryldata.vercel.app/integrations - we should have a few links that point at `/integrations`, that will need to get pointed elsewhere

alternatively, can we make `/integrations` just redirect to the new page?